rules/static.go

```diff
@@ -827,19 +827,20 @@ func (r *DefaultRuleRenderer) StaticRawPreroutingChain(ipVersion uint8) *Chain {
 		})
 	}
 
-	if ipVersion == 6 {
-		// Apply strict RPF check to packets from workload interfaces.  This prevents
-		// workloads from spoofing their IPs.  Note: non-privileged containers can't
-		// usually spoof but privileged containers and VMs can.
-		//
-		// We only do this for IPv6 because the IPv4 RPF check is handled via a sysctl.
-		// In addition, the IPv4 check is complicated by the fact that we have special
-		// case handling for DHCP to the host, which would require an exclusion.
-		rules = append(rules, Rule{
-			Match:  Match().MarkSingleBitSet(markFromWorkload).RPFCheckFailed(),
-			Action: DropAction{},
-		})
-	}
+	// Apply strict RPF check to packets from workload interfaces.  This prevents
+	// workloads from spoofing their IPs.  Note: non-privileged containers can't
+	// usually spoof but privileged containers and VMs can.
+	// But we also need to allow DHCP discover requests to be sent from the host.
+	// These DHCP requests will have source IP "0.0.0.0" and would otherwise be
+	// dropped by our RPF check rule.
+	rules = append(rules, Rule{
+		Match:  Match().SourceNet("0.0.0.0/0").DestNet("255.255.255.255").Protocol("udp").SourcePorts(68).DestPorts(67),
```

Comment:
The FV's are failing because the iptables is complaining: `host/network `255.255.255.255' not found`
Does anyone see what I'm missing?

Reply:
Nevermind, this isn't needed - we already have exceptions for OpenStack DHCP